config/dev.exs

```diff
@@ -23,3 +23,5 @@ config :archethic_playground, ArchethicPlaygroundWeb.Endpoint,
     esbuild: {Esbuild, :install_and_run, [:playground, ~w(--sourcemap=inline --watch)]},
     tailwind: {Tailwind, :install_and_run, [:default, ~w(--watch)]}
   ]
+
+config :archethic_playground, ArchethicPlaygroundWeb.DeployComponent, mainnet_allowed: true
```

Comment:
I think you should let environment variable but maybe you can use default value to true, otherwise we cannot override in dev with environment variable to test